apps/prairielearn/src/pages/instructorQuestionSettings/instructorQuestionSettings.html.ts

```diff
@@ -76,323 +76,265 @@ export function InstructorQuestionSettings({
+                        </button>
+                      `
+                    : ''
+                }
+                ${
+                  questionGHLink
+                    ? html`<a target="_blank" href="${questionGHLink}"> view on GitHub </a>`
+                    : ''
+                }
+                <input
+                  type="text"
+                  class="form-control"
+                  id="qid"
+                  name="qid"
+                  value="${resLocals.question.qid}"
+                  disabled
+                />
+                <small class="form-text text-muted">
+                  This is a unique identifier for the question. (e.g., "addNumbers")
+                </small>
+              </div>
+              <div class="form-group">
+                <label for="type">Type</label>
+                <input
+                  type="text"
+                  class="form-control"
+                  id="type"
+                  name="type"
+                  value="${resLocals.question.type}"
+                  disabled
+                />
+                <small class="form-text text-muted"> This is the type of question. </small>
+              </div>
+              <div class="form-group">
+                <div>Topic</div>
+                <div name="topic">
+                  ${renderEjs(__filename, "<%- include('../partials/topic') %>", {
+                    topic: resLocals.topic,
+                  })}
+                </div>
+              </div>
+              <hr />
+              <div class="form-group">
+                <div>Tags</div>
+                <div name="tags">
+                  ${renderEjs(__filename, "<%- include('../partials/tags') %>", {
+                    tags: resLocals.tags,
+                  })}
+                </div>
+              </div>
+              <hr />
+              <div class="form-group">
+                <div>
+                  Issues
+                  ${renderEjs(__filename, "<%- include('../partials/issueBadge') %>", {
+                    count: resLocals.open_issue_count,
+                    issueQid: resLocals.question.qid,
+                    suppressLink: resLocals.suppressLink,
+                    urlPrefix: resLocals.urlPrefix,
+                  })}
+                </div>
+              </div>
+              <hr />
+              <div class="form-group">
+                <div>Assessments</div>
+                <div>
+                  ${
+                    resLocals.assessments
                       ? renderEjs(__filename, "<%- include('../partials/assessments') %>", {
                           assessments: resLocals.assessments,
                           urlPrefix: resLocals.urlPrefix,
                         })
-                      : ''}
-                  </td>
-                </tr>
-                ${sharingEnabled
+                      : ''
+                  }
+                </div>
+              </div>
+              </form>
```

Comment:
I ended the form here. This is to prevent nested forms as we already have forms for public sharing and tests already on the page. However, I believe all of the fields that will want to be editable are captured in this form. That being said, is there a better way to do this? My concern here being that once we are ready to add a submit button, I am not sure where the best place is going to be for that.